source/NVDAObjects/JAB/__init__.py

```diff
@@ -1,3 +1,8 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2007-2021 NV Access Limited, Mozilla Corporation

```

Comment:
This is missing serval contributors from git blame, and I think "Mozilla Corporation" as a copy/paste mistake. However, now that the message is added, the list of contributors can be updated.

```suggestion
# Copyright (C) 2007-2021 NV Access Limited
```